ted2zim/scraper.py

```diff
@@ -168,31 +198,38 @@ def extract_all_video_links(self):
         self.update_title_and_description()
 
     def update_title_and_description(self):
-        if len(self.topics) > 1:
+        if self.playlist:
             if not self.title:
-                self.title = "TED Collection"
+                self.title = "TED Playlist"
```

Comment:
Playlists have a name and a description so that should be the default, if not provided.

Reply:
Done